### PR TITLE
[BE: HM-5] - Add MongoDB

### PR DESCRIPTION
**Pull request by ewgenij87snwork**
seeded db_full__2021-12-29_21-11
seeded db__2021-12-29_21-11

### CODE REVIEW

File: back/schemas/Posts.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    text: {
+      type: String,
+      required: [true, 'Text required']
+    },
+
+    author: {
+      type: Schema.Types.ObjectId,
+      required: [true, 'Author required'],
+      rel: 'Users'
```
Comment (urban-knight) [inline back/schemas/Posts.js:23]: Should be ref instead

**Review by afandul — APPROVED** [review]
Good solution

**Comment by ewgenij87snwork** [comment]
Thank you!

File: back/schemas/Posts.js (hunk 1 above)
Comment (ewgenij87snwork) [reply back/schemas/Posts.js:23]: @urban-knight Thank you!

**Pull request merged**